<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCCameraAdd</class>
 <widget class="QWidget" name="VSCCameraAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>881</width>
     <height>402</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="5" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Device Name</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineOnvifAddr">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEditPort">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEditRtspLoc">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonRtsp">
        <property name="text">
         <string>RTSP</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelIP">
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="labelOnvifProfileToken">
        <property name="text">
         <string>Profile Token</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="lineEditProfileToken">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QCheckBox" name="checkBoxProfileToken">
        <property name="text">
         <string>Token </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditName">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="labelRtspLoc">
        <property name="text">
         <string>RTSP Location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonOnvif">
        <property name="text">
         <string>ONVIF ProfileS</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditUser">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelUser">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditIP">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="labelOnvifAddr">
        <property name="text">
         <string>ONVIF Address</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonApply">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
